CONTRIBUTING.md

```diff
@@ -1,58 +1,56 @@
+# Contributing to ivadomed
 
+## Introduction
 
-# Contributing to IVADO Medical Imaging
+First off, thanks for taking the time to contribute!
 
-## Table of contents
-1. [Introduction](#introduction)
-2. [Opening an issue](#opening-an-issue)
-    1. [Before Submitting a New Issue](#before-submitting-a-new-issue)
-    2. [Submitting an Issue](#submitting-an-issue)
-3. [Contributing to IVADO Medical Imaging (Pull request)](#contributing-to-ivado-medical-imaging-pull-request)
-    1. [Opening a Branch](#opening-a-branch)
-    2. [Naming your Branch](#naming-your-branch)
-    3. [Developing](#developing)
-    4. [Committing](#committing)
-    5. [Submitting a Pull Request](#submitting-a-pull-request)
-4. [Versioning](#versioning)
+When contributing to this repository, please first discuss the change you wish
+to make via Github issue, email, or any other method with the core team before
```

Comment:
email: should we add an email address? alternatively "Github issue" could be enough

Reply:
indeed-- fixed